starters/vue3-xstate-css/tsconfig.json

```diff
@@ -10,5 +10,8 @@
     {
       "path": "./tsconfig.cypress-ct.json"
     }
-  ]
+  ],
+  "compilerOptions": {
+    "baseUrl": "./"
```

Comment:
This was the only small change Cypress needed, so I chose to wrap it into this PR.